torch/_inductor/graph.py

```diff
@@ -938,6 +938,9 @@ def compile_to_fn(self):
             code, linemap = self.codegen()
             output_code_log.debug("Output code: \n%s", code)
 
+            if config.cpp_wrapper:
```

Comment:
This will always be true because of `if self.aot_mode and self.cpp_wrapper`? I think can just get rid of `AotCodeCache.compile`.